src/FormExplore.js

```diff
@@ -1,11 +1,34 @@
 import React, { Component } from 'react';
 import Form from 'react-bootstrap/Form';
-import Button from 'react-bootstrap/Button'
+import Button from 'react-bootstrap/Button';
+
 export default class FormExplore extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            city: ''
+        }
+    }
+    handleInput = (e) => {
+        e.preventDefault();
+        this.setState({ city: e.target.value });
+      }
+      handleClick = (e) => {
+        e.preventDefault();
+        this.props.getLocation(this.state.city)
+    }
```

Comment:
Great work here!

It is helpful to put one line of whitespace between methods for easier readability and ensure that our open and closing curlies align 👍